vision/cloud-client/detect/detect.py

```diff
@@ -661,6 +661,95 @@ def detect_document_uri(uri):
 # [END vision_fulltext_detection_gcs]
 
 
+# [START vision_fulltext_detection_pdf]
+def detect_pdf_document(path):
+    """Detects document features in a PDF/TIFF/GIF file."""
+    from google.cloud import vision_v1p4beta1 as vision
+    client = vision.ImageAnnotatorClient()
+
+    with io.open(path, 'rb') as pdf_file:
+        content = pdf_file.read()
+
+    # Can be other like 'image/tiff' or 'image/gif'
+    mime_type = 'application/pdf'
+    file_bytes_input_config = vision.types.InputConfig(
+        content=content, mime_type=mime_type)
+
+    feature = vision.types.Feature(
+        type=vision.enums.Feature.Type.DOCUMENT_TEXT_DETECTION)
+    image_context = vision.types.ImageContext()
+    # Annotate the first two pages and the last one (max 5 pages)
+    # First page starts at 1, and not 0. Last page is -1.
+    pages = [1, 2, -1]
```

Comment:
Is that a max of 5 pages per PDF document?
I don't understand the API's pages, syntax.

Reply:
Yeah, took me a while to understand it too. 5 Pages per PDF; and you pass nonzero indices as a list.